service/src/main/java/bio/terra/workspace/service/resource/controlled/cloud/azure/database/ControlledAzureDatabaseResource.java

```diff
@@ -199,6 +199,7 @@ private void maybeSetupFederatedIdentity(FlightBeanBag flightBeanBag, ArrayList<
       steps.add(getManagedIdentityStep);
       steps.add(
           new GetFederatedIdentityStep(
+              getK8sNamespace(),
               getK8sNamespace(),
```

Comment:
Could you add a comment about why this is here twice?

Reply:
I ask only because when I first read the code, I assumed you had accidentally duplicated a line. But perhaps the name of the argument in `GetFederatedIdentityStep` is sufficient.